content/blog/2019-01-06-babette.md

```diff
@@ -0,0 +1,205 @@
+---
+slug: babette
+title: Call BEAST2 from R
+package_version: 2.0
+authors:
+  - Richèl J.C. Bilderbeek
+date: 2019-01-06
+categories: blog
+topicid: 
+tags:
+  - Software Peer Review
+  - R
+  - community
+  - software
+  - packages
+  - phylogeny
+  - phylogenetics
+  - BEAST
+  - BEAST2
+  - Bayesian
+---
+
+`babette` [^1] is a package to work with BEAST2 [^2], 
+a software platform for Bayesian evolutionary analysis from R.
+
+`babette` is a spin-off of my own academic research.
+As a PhD I work on models of diversification: mathematical descriptions
+of how species form new species. Instead of working on a species'
+individuals, I work on species as evolutionary lineages.
+A good way to show the evolutionary relationships between species
+are phylogenies. For example, this a phylogeny, with four species,
+eloquently named '1', '2', '3' and '4':
+
+![A phylogeny with four species](img/blog-images/2019-01-06-babette/phylogeny.png)
+
+In my research, I simulate thousands of these. I call these 
+phylogenies 'the truth'. Then I wonder, would such a phylogeny actually
+be true, how well would we be able to infer it? I say 'infer', instead
+of 'measure', as we cannot measure phylogenies directly in nature. 
+
+In nature, however, we can sequence the DNA of species. From these DNA
+sequences, we use software to neatly arrange these sequences, creating
+a DNA alignment. Such DNA alignments are where most researchers start from.
+Here is such an alignment, of four (eloquently named) species
+and their aligned DNA sequences, in which a different color represents a
+different nucleotide:
+
+![A DNA alignment of four species](img/blog-images/2019-01-06-babette/alignment.png)
+
+It is hard to only guess by eye which of the four species are most related.
+Luckily, we have software for such problems, for example, BEAST2.
+BEAST2 is a Bayesian phylogenetic tool that infers a posterior
+from a DNA alignment. The posterior contains multiple phylogenies,
+in which those that are more likely are present more often. Here is what these
+posterior trees look like when superimposed:
+
+![Posterior trees of four species](img/blog-images/2019-01-06-babette/densitree.png)
+
+Spoiler: actually, the phylogeny shown was the true phylogeny 
+underlying the alignment! You can see the inference has
```

Comment:
After spoiler, it would help to briefly say how we "can see the inference has problems recovering it". That is, note how the density of lines in the image is interpreted. (Hope I am not making this more complicated than it should be. You are welcome to disagree with this or any suggested changes)

Reply:
Agreed. I hope you enjoy the improvement. Feel free to make me clarify further if you think it would be better.